letra/_parser.py

```diff
@@ -1,12 +1,12 @@
-from schema import Optional, Regex, Schema, SchemaError
+from schema import Optional, Regex, Schema, SchemaError, Or
 from letra import Label
 
 color_regex = r"^([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$"
 
 label_schema = Schema(
     {
         "name": str,
-        Optional("description"): str,
+        Optional("description"): Or(str, None),
```

Comment:
This was an existing bug fixed as part of the broader changes in this PR. Apprently there can be a `null` value for the `description` key in the API response, but I'd originally been under the impression it would be an empty string in cases of labels missing descriptions